docs/modules/ROOT/pages/licensing.adoc

```diff
@@ -19,6 +19,9 @@ This software uses the following components with their associated licensing also
 |Grafana
 |link:https://github.com/grafana/grafana/blob/main/LICENSE[AGPL 3.0]
 
+|Grafana JSON API Plugin
+|link:https://github.com/marcusolsson/grafana-json-datasource/blob/main/
```

Comment:
You need the link text in square brackets, it looks funky: https://github.com/couchbaselabs/observability/blob/2482ac5a39531abdb06358745ef5c72a1cb8e63d/docs/modules/ROOT/pages/licensing.adoc

Reply:
That link also is 404 for me, we need to target the license file directly.

So https://github.com/marcusolsson/grafana-json-datasource/blob/main/LICENSE